api/tests/integration/AuthTest.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+use Tymon\JWTAuth\Token;
+use Tymon\JWTAuth\Payload;
+use Tymon\JWTAuth\Claims\Issuer;
+use Tymon\JWTAuth\Claims\IssuedAt;
+use Tymon\JWTAuth\Claims\Expiration;
+use Tymon\JWTAuth\Claims\NotBefore;
+use Tymon\JWTAuth\Claims\Audience;
+use Tymon\JWTAuth\Claims\Subject;
+use Tymon\JWTAuth\Claims\JwtId;
+use Tymon\JWTAuth\Claims\Custom;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class AuthTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    protected function callRefreshToken($token)
+    {
+        $this->get('/auth/jwt/refresh', [
+            'HTTP_AUTHORIZATION' => 'Bearer '.$token
```

Comment:
Can you add integration test with `Guzzle` to assert that the client can just use the plain `Authorization [...]` headers? I think there needs to be an update to the nginx config: `SetEnvIf Authorization .+ HTTP_AUTHORIZATION=$0`